src/components/CircularOutline/CircularOutline.js

```diff
@@ -62,7 +59,7 @@ class RoundedOutline extends Component<Props, State> {
   };
 
   wrapperNode: HTMLElement;
-  shapeNode: HTMLElement;
+  shapeNode: any;
```

Comment:
Were you getting a Flow error with `HTMLElement`? In general we want to avoid `any` types, since we lose the advantages of Flow.

It's possible the error was because `shapeNode` could be undefined, in which case the type would be `?HTMLElement`.

Reply:
Yes, it was working with my development environment (yarn start), but when I tried to commit the code, the ESLint hook was complaining that HTMLElement doesn't have a method "getTotalLength"
It is a bit of a code smell, let me try to get to the bottom of the issue.

Reply:
Ohh I see, right! That makes sense, since it's not a generic HTML element, it's an SVG Rect.

Yeah, you know what, `any` is fine. I checked the Flow codebase and couldn't find any type for SVG rects. Please add a comment over the type, something like "// Set to `any` because there is no proper type for SVG rects, and we need the method `getTotallength`"

Reply:
Well there is an interface "SVGPathElement" that has "getTotalLength()" and my editor understands what it is, but the pre-commit hook spits out the error that it's unresolved.

I will put this in a comment